client-common/src/tendermint/client.rs

```diff
@@ -25,9 +28,67 @@ pub trait Client: Send + Sync {
         heights: T,
     ) -> Result<Vec<BlockResults>>;
 
+    /// Make `validators` call to tendermint
+    fn validators(&self, _height: u64) -> Result<ValidatorsResponse> {
```

Comment:
There are several `MockClient` implementation in the test code, I find it tedious to replicate the panic instances there.

Reply:
Yes. Maybe you can just return `Ok(ValidatorsResponse::new())` instead of `panic!`?

Reply:
> Yes. Maybe you can just return `Ok(ValidatorsResponse::new())` instead of `panic!`? I'd ideally prefer not having a default implementation.

Actually, there are several methods are not used so far, I removed them, only keep `block_batch_verified` in the trait, now it's more feasible to remove the default implementation, which is done is new commit.